dwave/system/samplers/dwave_sampler.py

```diff
@@ -105,6 +105,12 @@ class DWaveSampler(dimod.Sampler, dimod.Structured):
             then it will instead propogate the `SolverNotFoundError` to the
             user.
 
+        order_by (callable/str/None):
+            Solver sorting key function or (or :class:`~dwave.cloud.Solver`
+            attribute/item dot-separated path).
+            See :class:`~dwave.cloud.Client.get_solvers` for a more detailed
+            description of the parameter.
+
```

Comment:
I've just realized https://github.com/dwavesystems/dwave-cloud-client/pull/405 broke the [design of `order_by`](https://github.com/dwavesystems/dwave-cloud-client/issues/407).

IMO, `order_by` should not be a parameter of `DWaveSampler`. I'll fix it in #317 after https://github.com/dwavesystems/dwave-cloud-client/issues/407 is fixed.

Reply:
Where should it be an argument?

Reply:
Part of solver definition, e.g. `solver=dict(hybrid=True, version__gt='1.0', order_by='-version')`.